src/chainstate/stacks/boot/pox_2_tests.rs

```diff
@@ -417,3 +417,324 @@ fn test_simple_pox_lockup_transition_pox_2() {
         "Bob tx1 should have resulted in a runtime error"
     );
 }
+
+/// In this test case, two Stackers, Alice and Bob stack and interact with the
```

Comment:
What does this method `test_pox_extend_transition_pox_2` test now? Does it test anything new? If this test was testing the last PR, then how does it exercise the new `.clar` code?

Reply:
Yes -- `test_pox_extend_transition_pox_2` covers the new functionality from the `stack-extend` function, in the function doc, it describes how Alice and Bob each interact with `stack-extend`. Will add a bit more detail to this comment.

Reply:
This was documented further in bbd826d4bde81aee6c26f134439b9d73d8885e34, and the test was updated to be a bit more clear in what it is doing.